rest_framework_simplejwt/views.py

```diff
@@ -116,5 +116,11 @@ class TokenBlacklistView(TokenViewBase):
 
     _serializer_class = api_settings.TOKEN_BLACKLIST_SERIALIZER
 
+    def post(self, request: Request, *args, **kwargs) -> Response:
+        response = super().post(request, *args, **kwargs)
+        serializer_data = response.data
+        serializer_data['Token detail'] = 'Token blacklisted'
```

Comment:
This should be changed to match a descriptive message such as that in the test, `Token detail` should be `message`.

Can you send in a fix?

```python
serializer_data['message'] = 'Token blacklisted'
```

Reply:
Hi thanks for your attention, sure as soon as possible